boards/nxp/mr_mcxn_t1/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2024-2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+zephyr_library()
+zephyr_library_sources(board.c)
+
+if(CONFIG_FLEXSPI_CONFIG_BLOCK_OFFSET)
```

Comment:
Remove anyting related to FlexSPI, we don't use the FlexSPI on the MR-MCXNT1 board.

Reply:
What about with new IO board? Or are those non-flex spi lines?